django_twilio/utils.py

```diff
@@ -91,5 +91,22 @@ def get_blacklisted_response(request):
     return None
 
 
+def create_sub_account(user, twilio_client, friendly_name=None):
+    new_account = twilio_client.api.accounts.create(friendly_name=friendly_name or user)
+    new_credential = Credential.objects.create(user=user, account_sid=new_account.sid, auth_token=new_account.auth_token, name=new_account.friendly_name)
+    return new_credential, new_account
+
+
+def close_sub_account(account_sid, twilio_client):
+    return twilio_client.accounts(account_sid).update(status="closed")
```

Comment:
Hi @jheld , 

I guess it should be also`twilio_client.api.accounts`?

Reply:
I did test this locally so I don't think it was breaking but if you'd like to make a PR to change it (or fix it), please do! I've just released `0.13.1b0` so there's a small window of time before `0.13.1` officially.